assets/app/data/crowbar/services/upgrade-status.factory.js

```diff
@@ -85,18 +85,25 @@
                             // on step failure trigger error handler
                             onError(response);
                         } else {
+                            // check for the case of postpone upgrade of compute nodes, after controller nodes were
```

Comment:
This is specific to the "nodes" step. Can you implement this inside the `onRunning()` callback on the last page and keep this service generic?

Reply:
Thanks for your explanation. That makes sense. I'll make the change.